### PR TITLE
cmake: Add build param to turn ON/OFF aklite-apps

### PR DESCRIPTION
**Pull request by mike-sul**
Add CMake build param to to turn ON/OFF the `aklite-apps` building.

### CODE REVIEW

**Review by quaresmajose — APPROVED** [review]

**Comment by caiotpereira** [comment]
LGTM

**Review by caiotpereira — APPROVED** [review]

**Comment by mike-sul** [comment]
@doanac Please review&approve (I can't merge it without your approval)

**Review by doanac — APPROVED** [review]

**Pull request merged**
